CLAUDE.md

```
@@ -1,51 +1,167 @@
+
+- **Flat and clean.** No gradients, no glow effects, no glass morphism. Subtle borders, sharp typography, monochrome with minimal accent colors for status indicators.
+- **Information-dense.** This is a war room, not a marketing page. Pack data in cleanly.
+- **Real-time feel.** Poll every 30s for fleet status, every 60s for PRs/activity. SSE later if needed.
+- **System fonts.** `-apple-system, BlinkMacSystemFont, 'SF Pro', system-ui, sans-serif`. Monospace for code/logs: `'SF Mono', 'Fira Code', monospace`.
+
+## Architecture
 
-```bash
-go build ./...
-./overmind
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)